src/store/actions.js

```diff
@@ -0,0 +1,10 @@
+import { formatDate } from "../utils/helpers/formatDate";
+
+export const addTask = (title) => ({
+  type: "ADD_TASK",
+  payload: {
+    id: Date.now(),
+    title,
+    createdDate: formatDate(new Date()),
```

Comment:
As previously mentioned, I suggest storing the date as Date instead of a string.